libs/components/forms/src/lib/modules/input-box/input-box-control.directive.ts

```diff
@@ -9,4 +12,20 @@ import { Directive } from '@angular/core';
     'input:not([skyId]):not(.sky-form-control),select:not([skyId]):not(.sky-form-control),textarea:not([skyId]):not(.sky-form-control)',
   standalone: true,
 })
-export class SkyInputBoxControlDirective {}
+export class SkyInputBoxControlDirective {
+  @HostBinding('autocomplete')
+  @Input()
+  public set autocomplete(value: string | undefined) {
+    this.#_autocomplete = value;
+    if (!value && this.#hostService) {
```

Comment:
If the `autocomplete` attribute is never specified, the default `off` value will never be applied. You'll either need to put the default value logic in the `get` accessor or set the default both in the `set` accessor and the private property declaration. This might be a place where it makes more sense to do the former.

```
@HostBinding('autocomplete')
@Input()
public set autocomplete(value: string | undefined) {
  this.#_autocomplete = value;
}

public get autocomplete(): string | undefined {
  return this.#_autocomplete || (this.#hostSvc ? 'off' : undefined);
}
```

Reply:
Done